packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -150,11 +150,64 @@
 				"value": 16
 			},
 			{
-				"label": "Basic Set CC",
+				"label": "Basic CC",
 				"value": 256
 			}
 		]
 	},
+	"binary_report_type": {
+		"label": "Motion Sensor Triggered Command",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Basic CC",
+				"value": 1
+			},
+			{
+				"label": "Binary Report CC",
```

Comment:
Binary Sensor or Binary Switch?

Reply:
Are these all reports or is it using `Basic CC Set` and `Binary Sensor(?) CC Report` for example?